src/coreclr/pal/src/exception/signal.cpp

```diff
@@ -582,6 +582,7 @@ Parameters :
 Return :
     The return value from the signal handler
 --*/
+__attribute__((noinline))
```

Comment:
Comment explaining the reason for noinline would be nice

Reply:
I will add one